assignments/lambda-classes.js

```diff
@@ -1 +1,256 @@
-// CODE here for your Lambda Classes
+//=====Global Functions=====
+
+function randomizeNumber(max) {
+	let posOrNeg = Math.random() < 0.2 ? -1 : 1;
+	let randomNumber = Math.floor(Math.random() * Math.floor(max));
+	return posOrNeg * randomNumber;
+}
+
+function gradingInstructor() {
+	instructorsAndProjectManagers = [];
+	instructorsAndProjectManagers.push(
+		ilana,
+		ibrahim,
+		ivor,
+		peter,
+		patrick,
+		penelope
+	);
+	return instructorsAndProjectManagers[Math.abs(randomizeNumber(5))];
```

Comment:
You need to keep minimum cognitive load on a line. maybe, separate out the return statement from randomiser and array.